pkg/deploy/registry/deploylog/rest.go

```diff
@@ -180,6 +186,31 @@ func (r *REST) Get(ctx kapi.Context, name string, opts runtime.Object) (runtime.
 	}, nil
 }
 
+// waitForExistingDeployment will use the timeout to wait for a deployment to appear.
+func (r *REST) waitForExistingDeployment(namespace, name string) (*kapi.ReplicationController, error) {
```

Comment:
All API requests have a timeout that is passed to them or defaulted (60s)
down from the apiserver.

On Wed, Jun 22, 2016 at 11:42 AM, Michail Kargakis <notifications@github.com
> wrote:

> In pkg/deploy/registry/deploylog/rest.go
> <https://github.com/openshift/origin/pull/9482#discussion_r68077934>:
>
> > @@ -180,6 +186,31 @@ func (r *REST) Get(ctx kapi.Context, name string, opts runtime.Object) (runtime.
> >  	}, nil
> >  }
> >
> > +// waitForExistingDeployment will use the timeout to wait for a deployment to appear.
> > +func (r *REST) waitForExistingDeployment(namespace, name string) (*kapi.ReplicationController, error) {
>
> We don't send any timeout currently. I have proposed such a thing in #5584
> (comment)
> <https://github.com/openshift/origin/issues/5584#issuecomment-160951829>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9482/files/53571d0332a00a9aa0df0515d34a867b65273f42#r68077934>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p6kM1IF6R56tlV_jtD3wdwjy4mtYks5qOVgBgaJpZM4I7sgr>
> .
>